.goreleaser.yml

```diff
@@ -0,0 +1,24 @@
+builds:
+  - binary: goprotoc
+    main: ./cmd/goprotoc
+    goos:
+      - linux
+      - darwin
+      - windows
```

Comment:
I'd be careful about this - I am not 100% sure that all the file operations that are done in `protoreflect` are compatible with Windows - maybe they are (you took pains to make sure we did `filepath.Rel` etc correctly) but before doing this I might set up testing for both this and `protoreflect` with AppVeyor

Reply:
Fair point. Although there's not yet any real tests, even for linux or OS X  😝 

Since I'm just planning on a v0.1 "pre-release" for now, I'm okay leaving it in for now (maybe it will even help solicit some feedback, even if the feedback is "this doesn't work right on Windows"). But that's a good point that some Windows testing will be needed before a real v1.0 (or remove it as a supported platform).